packages/backend/src/models/ContentModel/ContentConfigurations/SpaceContentConfiguration.ts

```diff
@@ -124,18 +129,16 @@ export const spaceContentConfiguration: ContentConfiguration<SpaceContentRow> =
                         );
                     }
 
-                    if (filters.spaceUuids) {
-                        void builder.whereIn(
-                            `${SpaceTableName}.space_uuid`,
-                            filters.spaceUuids,
-                        );
-                    }
-
                     if (filters.space?.parentSpaceUuid) {
                         void builder.where(
                             `${SpaceTableName}.parent_space_uuid`,
                             filters.space.parentSpaceUuid,
                         );
+                    } else if (filters.spaceUuids) {
+                        void builder.whereIn(
+                            `${SpaceTableName}.space_uuid`,
+                            filters.spaceUuids,
+                        );
```

Comment:
wdyt of this? I think it might become clearer when you work on your table...

Reply:
When does this happen?

Reply:
https://github.com/lightdash/lightdash/pull/14404/commits/47d3e82d49f0f739333307bb0adf4527d56880f0